stubs/docutils/docutils/io.pyi

```diff
@@ -1,5 +1,6 @@
-from _typeshed import OpenBinaryModeReading, OpenBinaryModeWriting, OpenTextModeReading, OpenTextModeWriting
-from typing import Any, ClassVar
+from _typeshed import OpenBinaryModeReading, OpenBinaryModeWriting, OpenTextModeReading, OpenTextModeWriting, SupportsWrite
+from typing import Any, ClassVar, Pattern
```

Comment:
Style nit: we prefer to import `Pattern` from `re` rather than `typing` in stub files :)

```suggestion
from re import Pattern
from typing import Any, ClassVar
```